src/cloudkit.cs

```diff
@@ -350,6 +555,52 @@ interface CKFetchRecordChangesOperation {
 			set; 
 		}
 	}
+	
+	[iOS (10,0), Watch (3,0), TV (10,0), Mac (12,10, onlyOn64 : true)]
+	[BaseType (typeof(CKDatabaseOperation))]
+	interface CKFetchRecordZoneChangesOperation
+	{
+		[Export ("initWithRecordZoneIDs:optionsByRecordZoneID:")]
+		IntPtr Constructor (CKRecordZoneID[] recordZoneIDs, [NullAllowed] NSDictionary<CKRecordZoneID, CKFetchRecordZoneChangesOptions> optionsByRecordZoneID);
+
+		[Export ("recordZoneIDs", ArgumentSemantic.Copy)]
+		CKRecordZoneID[] RecordZoneIds { get; set; }
+
+		[NullAllowed, Export ("optionsByRecordZoneID", ArgumentSemantic.Copy)]
+		NSDictionary<CKRecordZoneID, CKFetchRecordZoneChangesOptions> OptionsByRecordZoneId { get; set; }
```

Comment:
Let's just ensure all new ones are `ID`.

Adding `XAMCORE_4_0` just slow us down now, when we're on a time critical task to complete everything (for something that might never ship or is years away). A full audit will be needed anyway if/when we reach that point.